cmd/analyze.go

```diff
@@ -49,10 +50,8 @@ var analyzeCmd = &cobra.Command{
 }
 
 func checkAnalyzeArgNum(args []string) error {
-	var errMessage string
 	if len(args) != 1 {
-		errMessage = "'analyze' requires one image as an argument: container analyze [image]"
-		glog.Errorf(errMessage)
+		errMessage := "'analyze' requires one image as an argument: container analyze [image]"
```

Comment:
nit: don't even need to make this var here, just instantiate the new error with the string directly